### DISCUSSION TITLE
Artemis doesnt work now in fire tv

### DISCUSSION THREAD

**Discussion by Jmlp93** [ClassicOldSong/moonlight-android#66] [General]
Previous version works well, but the newest doesnt work, why?

**Comment by ClassicOldSong** [comment]
Which version works and which doesn't?

Can you post the logcat when the app crashes? I don't have a Fire TV to test but there shouldn't be changes that break basic functionality.

**Reply by Jmlp93** [reply]
When I have time, i will send you de logcat. I don't actually know if the app works or not. The app opens, but when I click on settings, it closes. On my mobile device, it does work.

**Reply by Jmlp93** [reply]
I dont know how to find de logcat.
I have installed the last version, but it continue crashing when i press settings

**Comment by ClassicOldSong** [comment]
Should have fixed in https://github.com/ClassicOldSong/moonlight-android/releases/tag/v12.1.241203

**Comment by ClassicOldSong** [comment]
That version still doesn't work, try the latest: https://github.com/ClassicOldSong/moonlight-android/releases/tag/v12.1.241203-2

**Reply by Jmlp93** [reply]
Thanks, now it works.
